packages/ckeditor5/src/plugin.js

```diff
@@ -257,6 +257,14 @@ export default class MathType extends Plugin {
       // And obtain the complete formula
       formula = Util.htmlSanitize(`<math${mathAttributes}>${formula}</math>`);
 
+      // Replaces the < & > characters to its HTMLEntity to avoid render issues.
+      formula = formula.split('"<"').join('"&lt;"')
```

Comment:
Instead of `split` and `join`, it might be more efficient and readable to use `replace`.

Reply:
Replace seems to be faster than split+join, it should be interesting to change this, event though it may not be much of a performance improvement